server/test/incident.js

```diff
@@ -21,604 +23,500 @@ const notFound = `/api/v1/red-flag/${notFoundRedFlagId}`;
-                    expect(res.body.status).to.equal(400);
-                    res.body.message.should.be.a('string');
-                    done();
-                });
-        });
-        it('Should return 400 status code for empty comment ', (done) => {
-            chai.request(app)
-                .post(url)
-                .send(inValidComment[1])
-                .end((err, res) => {
-                    res.should.have.status(400);
-                    res.body.should.be.a('object');
-                    expect(res.body.status).to.equal(400);
-                    res.body.message.should.be.a('string');
-                    done();
-                });
-        });
-        it('Should return 400 status code for comment not a string', (done) => {
-            chai.request(app)
-                .post(url)
-                .send(inValidComment[2])
-                .end((err, res) => {
-                    res.should.have.status(400);
-                    res.body.should.be.a('object');
-                    expect(res.body.status).to.equal(400);
-                    res.body.message.should.be.a('string');
-                    done();
-                });
-        });
-        it('Should return 400 status code for comment too short ', (done) => {
-            chai.request(app)
-                .post(url)
-                .send(inValidComment[3])
-                .end((err, res) => {
-                    res.should.have.status(400);
-                    res.body.should.be.a('object');
-                    expect(res.body.status).to.equal(400);
-                    res.body.message.should.be.a('string');
-                    done();
-                });
-        });
-       
-        /**
-      * **********************************
-      * Test for comment ---- ends here!
-      * **********************************
-      */
-        /**----------------------------------------------------- */
-        /**
-        * **********************************
-        * Test Create incident --- begins here
-        * **********************************
-        */
-
-
-        it('Should return 404 status code if incident already exists', (done) => {
-            chai.request(app)
-                .post(url1)
-                .send(validIncident[1])
-                .end((err, res) => {
-                    res.should.have.status(404);
-                    res.body.should.be.a('object');
-                    expect(404);
-                    res.body.message.should.be.a('string');
-
-                    done();
-                });
+    });
+    it('Should return 400 status code for incident date not a string', (done) => {
+      chai.request(app)
+        .post(url)
+        .send(inValidIncidentDate[1])
+        .end((err, res) => {
+          res.should.have.status(400);
+          res.body.should.be.a('object');
+          expect(res.body.status).to.equal(400);
+          res.body.message.should.be.a('string');
```

Comment:
Please test that the content of the message matches requirements.

Reply:
Acknowledged and fixed